plugins/openpgp/src/account_settings_entry.vala

```diff
@@ -69,7 +69,7 @@ public class AccountSettingsEntry : Plugins.AccountSettingsEntry {
             return;
         }
         if (keys.size == 0) {
-            label.set_markup(build_markup_string(_("Key publishing disabled"), _("No keys available. Generate one!")));
+            label.set_markup(build_markup_string(_("Key publishing disabled"), _("No keys available. Generate one or check if your keys aren't expired or revoked!")));
```

Comment:
You also need to fix the gettext files, at least for Russian, English and German.

Reply:
Okay. I'll update the dino.pot file and fr.po file at least, I don't know German and Russian that much.